tests/dummy/translations/en-us.json

```diff
@@ -0,0 +1,37 @@
+{
+  "webrtcTroubleshoot": {
```

Comment:
Is there a way to have the dummy app load translations from the addon instead of duplicating strings?